spec/code_climate/test_reporter/git_spec.rb

```diff
@@ -79,5 +79,18 @@ def self.root
       end
     end
 
+    describe 'committed_at_from_git_or_ci' do
+      it 'returns the committed_at from git' do
+        expect(Git.committed_at_from_git_or_ci).to eq Git.send(:committed_at_from_git)
+      end
+
+      it 'returns the committed_at from git if there is no ci committed_at' do
```

Comment:
looks like this scenario description should now be "returns the committed_at from ci if there is no git committed_at"